open-metadata-implementation/integration-services/search-integrator/search-integrator-api/src/main/java/org/odpi/openmetadata/integrationservices/search/connector/SearchIntegratorContext.java

```diff
@@ -0,0 +1,48 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+
+package org.odpi.openmetadata.integrationservices.search.connector;
+
+import org.odpi.openmetadata.frameworks.auditlog.AuditLog;
+
+/**
+ * LineageIntegratorContext provides a wrapper around the Asset Manager OMAS client.
```

Comment:
Should LineageIntegratorAPI be SearchIntegratorAPI

Reply:
updated